internal/file/file_manager_service.go

```diff
@@ -193,13 +211,51 @@ func (fms *FileManagerService) ConfigApply(ctx context.Context,
 	return model.OK, nil
 }
 
+func (fms *FileManagerService) RollbackTempFiles(ctx context.Context) error {
```

Comment:
```suggestion
func (fms *FileManagerService) backupFiles(ctx context.Context) error {
```
I think this might be a more descriptive name for this function. Also I dont think it needs to be a public function